### DISCUSSION TITLE
Daily 20/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#181] [General]
Ayer levanté una instancia de AWS para correr lo que hice de FastAPI y modifiqué los formularios del sitio para que envíen la información allá y reciban y muestren la respuesta. También creé una discusión para que vayan preparándose sobre qué es lo que quieren hablar en el video final. Hoy hay clases de SGC y full-stack, también voy a considerar armar pruebas unitarias para el resto de los modelos y controladores, también viendo cómo pueden hacer para agregar código a esas clases para figurar en los commits.

**Comment by AhumadaCeleste** [comment]
Ayer Termine de estudiar el material de boobstrap y wordpress. Intente continuar en practica con local y wordpress pero mi conexión estaba inestable y no me abrió el sitio. Lo intentare nuevamente hoy. Quería finalizar un modelo, minimalista, para proponer según acordamos el domingo. También complete la lección de inglés al finalizar la clase. Me interiorice un poco sobre las funcionalidades de JavaScrit para proponer user histories para comenzar a considerar lo que incluiremos en el próximo sprint. Hoy voy a repasar para el examen de proyecto, e iniciare con los apuntes de java. SI tengo tiempo avanzar con otro vídeo del prof. Mainero para utilizar el fin de semana en iniciar el ultimo modulo de Python, para utilizar el fin de semana en iniciar el ultimo modulo de Python. Mañana veré las clase de hoy. Ahorro tiempo viéndolas aceleradas. y espero completar JavaScript

**Comment by NataliaAlvarezIspc** [comment]
Ayer avance con los contenidos de GP, me quedan algunos temas para repasar. También estuve estudiando los apuntes de Java, complete todo el apunte que se encuentra en la plataforma.
Hoy estaré con Wordpress y con lo que me queda de GP. Por la noche participaré de las clases sincrónicas de SGC y Full.

**Comment by Brian-Almada** [comment]
Ayer pude avanzar con ingles (estudio y actividades). Ahora estoy escuchando la clase de SGC. Cuando termine voy a repasar para la valoración de PI.
